controllers/account_controller.go

```diff
@@ -26,835 +26,822 @@ SOFTWARE.
 package controllers
 
 import (
-	"context"
-	"fmt"
-
-	"github.com/nats-io/jwt/v2"
-	"github.com/nats-io/nats.go"
-	"github.com/nats-io/nkeys"
-	"github.com/versori-oss/nats-account-operator/api/accounts/v1alpha1"
-	"github.com/versori-oss/nats-account-operator/controllers/resources"
-	accountsclientsets "github.com/versori-oss/nats-account-operator/pkg/generated/clientset/versioned/typed/accounts/v1alpha1"
-	"github.com/versori-oss/nats-account-operator/pkg/helpers"
-	"github.com/versori-oss/nats-account-operator/pkg/nsc"
-	"go.uber.org/multierr"
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/equality"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/types"
-	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
-	"sigs.k8s.io/controller-runtime/pkg/handler"
-	"sigs.k8s.io/controller-runtime/pkg/log"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-	"sigs.k8s.io/controller-runtime/pkg/source"
+    "context"
+    "fmt"
```

Comment:
Why is this whole file reformatted? 

Also it seems to follow some different rules compared to controllers/base_controller.go